cmd/frontend/graphqlbackend/search.go

```diff
@@ -208,6 +257,7 @@ func (r *searchResolver) countIsSet() bool {
 }
 
 const defaultMaxSearchResults = 30
+const maxSearchResultsPerPaginationRequest = 5000
```

Comment:
```suggestion
const maxSearchResultsPerPaginatedRequest = 5000
```

nit: worded as-is this wording confuses me a bit; In my mind "paginated request" == "request that has been or is paginated", while "pagination request" == "a request that something become paginated"

Reply:
Makes sense, I wasn't sure of this either. I initially called this '..PerPage..` but that might make unfamiliar folks think this refers to the actual pages being rendered in the webapp, whereas it's specific to the pagination functionality.